runtime/src/accounts_db.rs

```diff
@@ -3703,12 +3703,11 @@ impl AccountsDb {
 
         let mut rewrite_elapsed = Measure::start("rewrite_elapsed");
         let mut dead_storages = vec![];
-        let mut find_alive_elapsed = 0;
-        let mut create_and_insert_store_elapsed = 0;
-        let mut write_storage_elapsed = 0;
+        let mut create_and_insert_store_elapsed_us = 0;
+        let mut write_storage_elapsed = Measure::start("empty_by_default");
```

Comment:
the different timing patterns here seem like an odd choice:
1. `create_and_insert_store_elapsed_us`: we use an int to accumulate time
2. `write_storage_elapsed`: initialize a dummy measure, overwrite it and stop if necessary, **relies on not calling stop to mean 0**.
3. `find_alive_elapsed`: measures a chunk of code.

Would be preferrable to have them all be consistent.

Reply:
Alternatively, just make `write_storage_elapsed` an option - that seems cleaner.

Reply:
I've never seen us use option and for an option I can't unwrap a default. int _us seems cleaner.